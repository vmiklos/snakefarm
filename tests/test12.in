out 1
time 1
load test12.map
addplayer 1
addsnake 1 1 0 1 1 0
11 0
10 0
9 0
endsnake
addplayer 2
addsnake 2 2 0 1 1 0
17 0
16 0
endsnake
comment # csak 1 egyseg a jatekido, tehat itt le kell allnia, es az elso jatekos kigyoja a leghosszabb, tehat o a nyertes
step
exit
