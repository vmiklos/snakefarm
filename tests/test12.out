Event MapLoad test12.map
# csak 1 egyseg a jatekido, tehat itt le kell allnia, es az elso jatekos kigyoja a leghosszabb, tehat o a nyertes
Event Step 1
End Step
Event Step 2
End Step
Event Win 1
Event Exit
